index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-0GGDWZYGDE"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        window.dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "XXXXX");
+    </script>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Capstone</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body class="index-body">
+    <div class="optionsheadercontainer">
+    <ul class="optionsheader">
+      <li><i><img class="icon1" src="./imgs/facebook-icon-png-774.png" alt="" href="#"></i></li>
+      <li><i><img class="icon2" src="./imgs/Twitter.png" alt="" href="#"></i></li>
```

Comment:
- [ ] A good practice for `<img>` tag is always add a value for `alt` tag, please add it. 🐻